src/main/java/frc/robot/Auto.java

```diff
@@ -216,7 +207,8 @@ public CommandBase combinedAuto(ConeStation conePos, CubePickupLocation cubePos,
                 conePos.communityPosition.inCommunity.getPath(drive),
                 scoreCubeCmd,
                 scoreCube()
-        // position to be in front of drive station in community? I'll add that
+        // position to be in front of drive station in community? I'll add that <- ha
+        // you thought, no becuase this auto is useless
```

Comment:
If we're going to be snarky, can we be snarky with the correct spelling of "because"?